climada/hazard/centroids/centr.py

```diff
@@ -325,7 +325,7 @@ def set_lat_lon(self, lat, lon, crs=DEF_CRS):
             CRS. Default: DEF_CRS
         """
         self.__init__()
-        self.lat, self.lon, self.geometry = lat, lon, gpd.GeoSeries(crs=crs)
+        self.lat, self.lon, self.geometry = np.array(lat), np.array(lon), gpd.GeoSeries(crs=crs)
```

Comment:
Note that `np.array(lat)` creates a copy if `lat` is an array already. Is this what we want? An alternative would be `np.asarray`.

Reply:
that probably makes more sense!

Reply:
updated